content/pages/meetup/2023/0920-nycu.rst

```diff
@@ -0,0 +1,105 @@
+========================================
+Meetup 2023 September 20th in NYCU
```

Comment:
Could you please include the training name "do-unknown-things" and make it like:

```
Meetup 2023 September 20th in NYCU (do-unknown-things training)
```